### PR TITLE
New bikeshed version

### PR DESCRIPTION
**Pull request by plehegar**
Use the latest bikeshed (now frozen to https://github.com/tabatkins/bikeshed/commit/fbf1456a756299b3ff6d248d0857ec87f2e68cd7 )

### CODE REVIEW

**Comment by plehegar** [comment]
The 2 remaining link errors will need to be fixed later. show event does not (no longer?) exist in HTML. Web Worker is missing a definition.
Otherwise, this should be good to go. Additional refinements can be done to improve the cross-spec linking but that's for another day...
@chaals @LJWatson don't forget to update your bikeshed and run bikeshed update if you're going to build the spec locally.

**Comment by plehegar** [comment]
@siusin let me know if I forgot anything...

File: sections/semantics-forms.include [hunk 1]
```diff
@@ -6436,7 +6436,7 @@ You cannot submit this form when the field is incorrect.</samp></pre>
       </td>
      </tr>
      <tr>
-      <th><code>words</code></th>
+      <th><dfn attr-value for="input/autocapitalize"><code>words</code></a></th>
```
Comment (chaals) [inline sections/semantics-forms.include]: the `</a>` should be `</dfn>`

File: sections/semantics-forms.include [hunk 2]
```diff
@@ -6445,7 +6445,7 @@ You cannot submit this form when the field is incorrect.</samp></pre>
       </td>
      </tr>
      <tr>
-      <th><code>sentences</code></th>
+      <th><dfn attr-value for="input/autocapitalize"><code>sentences</code></a></th>
```
Comment (chaals) [inline sections/semantics-forms.include]: the `</a>` should be `</dfn>`

**Review by chaals — CHANGES_REQUESTED** [review]
minor tweaks. I should be able to make these in a couple of minutes.

**Review by chaals — APPROVED** [review]

**Pull request merged**